spec/commands/exec_spec.rb

```diff
@@ -6,6 +6,22 @@
     system_gems(system_gems_to_install, :path => :bundle_path)
   end
 
+  it "works with --gemfile flag" do
+    create_file "CustomGemfile", <<-G
```

Comment:
Thinking more about this test, do you need 2 different Gemfiles?

Reply:
I did that to ensure that the environment variable was set correctly both the times and `rackup` produced the version corresponding to the mentioned gemfiles.

Reply:
You're already testing `bundle exec` with the `--gemfile` option. So testing it again is redundant, can you please remove it.